src/main/java/pl/javastart/voting/VotingResult.java

```diff
@@ -1,9 +1,19 @@
 package pl.javastart.voting;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * Możesz dodać kolejne metody i pola do klasy. Nie zmieniaj istniejących metod.
  */
+
 public class VotingResult {
+    private static final String VOTE_FOR_PASSAGE = "za";
+    private static final String VOTE_AGAIN = "przeciw";
+    private static final String VOTE_ABSTENTION = "wstrzymał się";
+    private static final int MULTIPLAYER = 100;
+
+    List<Vote> votes = new ArrayList<>();
```

Comment:
private

Reply:
nadal do poprawy

Reply:
Byłem święcie przekonany, że to zmieniłem. Może pózniej za daleko poleciałem z ctrl + z po późniejszych poprawkach. anyway, poprawione